src/index.ts

```diff
@@ -97,6 +100,30 @@ function buildHtml(
     containerElement.querySelector(`#${ns}footer`)!.innerHTML =
       mergeConfig.footer;
   }
+
+  // Enable image upload
+  if (mergeConfig.features?.includes("upload-image")) {
+    const sizeLimitMB = 4;
```

Comment:
Set size limit to 4MB. Might add support for custom size limit later (if needed).